src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -376,7 +374,7 @@ private void addTaskArgument(ArgumentListBuilder args) {
 
   @VisibleForTesting
   void populateConfiguration(ExtendedArgumentListBuilder args, Run<?, ?> build, FilePath workspace,
-    TaskListener listener, EnvVars env, @Nullable SonarInstallation si) throws IOException, InterruptedException {
+    TaskListener listener, EnvVars env, SonarInstallation si) throws IOException, InterruptedException {
     if (si != null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)